src/Class Specs/Class_sport.jsx

```diff
@@ -313,18 +291,20 @@ class Sport extends React.Component {
 
     if (found == false) {
       this.Graphs.Team_players(name, this.year).then((result) => {
-        //console.log(result);
         this.teams_player[0].push(`${name}${this.year}`);
-        data = this.format_players(result);
-        this.teams_player[1].push(data);
-        this.Select_team = `${name}`;
-        this.set_co(this.Select_team);
-        console.log("done");
+        this.format_players(result).then((result) => {
+          this.teams_player[1].push(result);
+          this.Select_team = `${name}`;
+          this.set_co(name);
+          console.log("done");
+          if (this.s_c_e == true) {
+            this.search_co_event(name + "1q");
+          }
```

Comment:
From what this looks like, it seems that when the elements loaded with the fetch api, it did not refresh the search_co_event, which was used to reverify elements or the useeffect, since the Useeffect only applied or worked with functions from the useState.

Reply:
That is true, in which it was better to change the search co event to save the name to the useState, but the "1q" was used to make the save string different.